docs/ADD_NEW_EXCHANGE.md

```diff
@@ -0,0 +1,1062 @@
+       },
+       "configFormat": {
+        "uppercase": true,
+        "delimiter": "-"
+       }
+      },
+      "spot": {
+       "enabled": "BTC/USD",
+       "available": "BTC/USD",
+       "requestFormat": {
+        "uppercase": true,
+        "delimiter": "/"
+       },
+       "configFormat": {
+        "uppercase": true,
+        "delimiter": "/"
+       }
+      }
+     }
+    },
+   "api": {
+    "authenticatedSupport": false,
+    "authenticatedWebsocketApiSupport": false,
+    "endpoints": {
+     "url": "NON_DEFAULT_HTTP_LINK_TO_EXCHANGE_API",
+     "urlSecondary": "NON_DEFAULT_HTTP_LINK_TO_EXCHANGE_API",
+     "websocketURL": "NON_DEFAULT_HTTP_LINK_TO_WEBSOCKET_EXCHANGE_API"
+    },
+    "credentials": {
+     "key": "Key",
+     "secret": "Secret"
+    },
+    "credentialsValidator": {
+     "requiresKey": true,
+     "requiresSecret": true
+    }
+   },
+   "features": {
+    "supports": {
+     "restAPI": true,
+     "restCapabilities": {
+      "tickerBatching": true,
+      "autoPairUpdates": true
+     },
+     "websocketAPI": true,
+     "websocketCapabilities": {}
+    },
+    "enabled": {
+     "autoPairUpdates": true,
+     "websocketAPI": false
+    }
+   },
+   "bankAccounts": [
+    {
+     "enabled": false,
+     "bankName": "",
+     "bankAddress": "",
+     "bankPostalCode": "",
+     "bankPostalCity": "",
+     "bankCountry": "",
+     "accountName": "",
+     "accountNumber": "",
+     "swiftCode": "",
+     "iban": "",
+     "supportedCurrencies": ""
+    }
+   ]
+  },
+```
+
+#### Available pairs will be automatically filled in the configs when wrapper functions are filled out and gocryptotrader is run with the new exchange enabled
+
+### Add the currency structs in ftx_wrapper.go:
+
+#### Futures currency support:
+
+Similar to the configs, spot support is inbuilt but other asset types will need to be manually supported
+
+```go
+	spot := currency.PairStore{
+		RequestFormat: &currency.PairFormat{
+			Uppercase: true,
+			Delimiter: "/",
+		},
+		ConfigFormat: &currency.PairFormat{
+			Uppercase: true,
+			Delimiter: "/",
+		},
+	}
+	futures := currency.PairStore{
+		RequestFormat: &currency.PairFormat{
+			Uppercase: true,
+			Delimiter: "-",
+		},
+		ConfigFormat: &currency.PairFormat{
+			Uppercase: true,
+			Delimiter: "-",
+		},
+	}
+```
+
+### Document the addition of the new exchange (FTX exchange is used as an example below):
+
+#### root Readme.md:
+```go
+| Exchange | REST API | Streaming API | FIX API |
+|----------|------|-----------|-----|
+| Alphapoint | Yes  | Yes        | NA  |
+| Binance| Yes  | Yes        | NA  |
+| Bitfinex | Yes  | Yes        | NA  |
+| Bitflyer | Yes  | No      | NA  |
+| Bithumb | Yes  | NA       | NA  |
+| BitMEX | Yes | Yes | NA |
+| Bitstamp | Yes  | Yes       | No  |
+| Bittrex | Yes | No | NA |
+| BTCMarkets | Yes | No       | NA  |
+| BTSE | Yes | Yes | NA |
+| COINUT | Yes | Yes | NA |
+| Exmo | Yes | NA | NA |
+| NewExampleName | Yes | Yes | No | // <-------- 
```

Comment:
Add in that Yes means supported and implemented, No means supported and not yet implemented and NA means protocol unsupported.